### DISCUSSION TITLE
第2节课笔记(16班)

### DISCUSSION THREAD

**Discussion by Yang-Changhui** [InternLM/Tutorial#117] [笔记]
16班第二次笔记链接请提交在下面评论

**Comment by Eadral** [comment]
https://www.yuque.com/eadral/ruaugi/fdfqzgx2mdvyegfp?singleDoc

**Comment by EnableAsync** [comment]
https://enableasync.github.io/internlm/internlm-02

**Comment by Aitejiu** [comment]
https://www.cnblogs.com/aitejiu/p/17948025

**Comment by Star-cre** [comment]
课堂公开笔记：https://www.bilibili.com/video/BV1Ci4y1z72H/?vd_source=2fb638751797274bd22bea982387a179
ID：戈壁的老王
csdn博客：https://zhilengnuan.blog.csdn.net/article/details/135386192

**Comment by future-2002** [comment]
https://blog.csdn.net/Y2483490891/article/details/135424276?spm=1001.2014.3001.5501

**Comment by etheral12138** [comment]
https://y1yc0dwhb0o.feishu.cn/docx/NbpHdvYdKoQVH3xHOlxchq7CnCh?from=from_copylink

**Comment by 2404589803** [comment]
https://2404589803.github.io/My-datawhale-learning/#/shusheng_puyu_learnning53/ch02

**Comment by morcake** [comment]
https://blog.51cto.com/morcake/9105322
https://blog.51cto.com/morcake/9105636
https://blog.51cto.com/morcake/9118470

**Comment by Durian-1111** [comment]
**Day02-LearningNote**
链接：https://xby48gs45u.feishu.cn/docx/XlNidoEAJoxmzGxQo42c6TQRngg?from=from_copylink

**Comment by andrepan** [comment]
https://www.yuque.com/andrepan/eg7adh/sof250cnu7u355at?singleDoc# 《第二课》

**Comment by TianCai19** [comment]
https://codetwice.notion.site/b1699af4914a44cc94800403723e1e5c?pvs=4

**Comment by hhhanqian** [comment]
89 【书生·浦语大模型笔记（二） - 嗨嗨 | 小红书 - 你的生活指南】 😆 IOA6sxKT3zhCHVI 😆 http://xhslink.com/WpdQXy

**Comment by Dhaizei** [comment]
http://t.csdnimg.cn/65pNm

**Comment by Aorg** [comment]
https://github.com/Aorg/Assignment/blob/main/lesson2_note.md

**Comment by springkin** [comment]
https://airy-planarian-ede.notion.site/Demo-f0364de6d185498899cca879bef1b5e4?pvs=4

**Comment by zszbyzsz** [comment]
https://juejin.cn/post/7321526403433922595

**Comment by yechenzhi** [comment]
https://docs.qq.com/doc/DZFZMU1VLUmt0emZt

**Comment by xuanxiaochenmi** [comment]
https://github.com/xuanxiaochenmi/InternLLM_notes/blob/master/lesson_2/lesson_2.md
